.prettierrc

```diff
@@ -1,5 +1,7 @@
 {
```

Comment:
Maybe consolidate these into the eslint rules in package.json?

Reply:
that's need some extra configuration to work together, I suggest to create task